doc/source/install/cmake_options.rst

```diff
@@ -0,0 +1,89 @@
+.. -*- rst -*-
+
+CMake options
+=============
+
+This section describes important options of CMake.
+
+To get more details about installing Groonga from the source with CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+``-G GENERATOR``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a generator.
+
+This is a mandatory option.
```

Comment:
It's optional.

Reply:
Ah, I have misunderstood.

The CMake document says it falls back to a builtin default selection if we don't specify this parameter.
What is the builtin default selection in this case?
(Which generator do we use?)

https://cmake.org/cmake/help/latest/manual/cmake.1.html

> If not specified, CMake checks the [CMAKE_GENERATOR](https://cmake.org/cmake/help/latest/envvar/CMAKE_GENERATOR.html#envvar:CMAKE_GENERATOR) environment variable and otherwise falls back to a builtin default selection.

Reply:
It depends on the system. "Unix Makefiles" is the one in the following system:

```console
$ cmake --help
...
The following generators are available on this platform (* marks default):
  Green Hills MULTI            = Generates Green Hills MULTI files
                                 (experimental, work-in-progress).
* Unix Makefiles               = Generates standard UNIX makefiles.
  Ninja                        = Generates build.ninja files.
  Ninja Multi-Config           = Generates build-<Config>.ninja files.
  Watcom WMake                 = Generates Watcom WMake makefiles.
  CodeBlocks - Ninja           = Generates CodeBlocks project files.
  CodeBlocks - Unix Makefiles  = Generates CodeBlocks project files.
  CodeLite - Ninja             = Generates CodeLite project files.
  CodeLite - Unix Makefiles    = Generates CodeLite project files.
  Eclipse CDT4 - Ninja         = Generates Eclipse CDT 4.0 project files.
  Eclipse CDT4 - Unix Makefiles= Generates Eclipse CDT 4.0 project files.
  Kate - Ninja                 = Generates Kate project files.
  Kate - Unix Makefiles        = Generates Kate project files.
  Sublime Text 2 - Ninja       = Generates Sublime Text 2 project files.
  Sublime Text 2 - Unix Makefiles
```

Reply:
I see. Thanks!!

Reply:
Fixed.